docs-examples/example-java/src/main/resources/application.yml

```diff
@@ -0,0 +1,16 @@
+micronaut:
+  application:
+    name: java
+  server:
+    port: 8081
+pulsar:
+  enabled: true
+  service-url: pulsar://localhost:6650
+  oauth-issuer-url: http://localhost:9090/auth/realms/master
+  oauth-credentials-url: file://src/main/resources/credentials.json

```

Comment:
is it possible to load this from the classpath instead of using a file URL? I assume this syntax will cause problems in GraalVM and/or in Docker

Reply:
Well the client library uses such paths so I'm not aware on how to avoid it except from exposing HTTP endpoint which would respond to only localhost to localhost. Then it would serve file content. Otherwise I'll require more testing on how to trick Pulsar Java client to use my things. It actually accepts URL that get's converted into Pulsars URL then back again to java URL. It's really strange library in some cases but I'll give it my best to avoid this.